src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientTest.java

```diff
@@ -50,13 +50,8 @@
 import net.snowflake.ingest.streaming.OpenChannelRequest;
 import net.snowflake.ingest.streaming.SnowflakeStreamingIngestClient;
 import net.snowflake.ingest.streaming.SnowflakeStreamingIngestClientFactory;
-import net.snowflake.ingest.utils.ErrorCode;
-import net.snowflake.ingest.utils.Pair;
-import net.snowflake.ingest.utils.ParameterProvider;
-import net.snowflake.ingest.utils.SFException;
-import net.snowflake.ingest.utils.SnowflakeURL;
-import net.snowflake.ingest.utils.Utils;
-import org.apache.arrow.vector.VectorSchemaRoot;
+import net.snowflake.ingest.utils.*;
```

Comment:
seems like you're still using *, do you want to update your intelliJ settings?

Reply:
sorry leftover, I changed, also enabled warning on this.